src/features/device-config/device-config.ts

```diff
@@ -120,6 +121,8 @@ export const generateConfig = async (
 				delta: `https://${DELTA_HOST}`,
 				registry: REGISTRY2_HOST,
 				vpn: VPN_HOST,
+				// If undefined, balena-device-config won't generate a logs endpoint
+				logs: LOGS_HOST ? `https://${LOGS_HOST}` : undefined,
```

Comment:
If you want to check for nullish then I prefer to be explicit about it (it can avoid confusion later):
```suggestion
				logs: LOGS_HOST != null ? `https://${LOGS_HOST}` : undefined,
```

Reply:
Updated in https://github.com/balena-io/open-balena-api/commit/006d7e85a29fab30f5e44589e0f6448c6e4a6a82 👍🏼